enable-ius.sh

```diff
@@ -3,6 +3,12 @@
 # Script to setup the IUS public repository on your EL server.
 # Tested on CentOS/RHEL 6/7.
 
+set -o errexit
+if [[ ${UID} -ne 0 && ${EUID} -ne 0 ]]; then
```

Comment:
`EUID` is only different from `UID` when a non-root user executes a file owned by root with the setuid bit.  I don't see that as a realistic scenario for this script, so let's check for `UID` only.

Reply:
Done.